From 25f1bebf1bc4c8da47f976d24a7a424253744e2e Mon Sep 17 00:00:00 2001
From: Alex Kiernan <alex.kiernan@gmail.com>
Date: Wed, 24 Mar 2021 09:10:17 +0000
Subject: [PATCH] libcoap: Fix gnu-configize error

Fix:

  autoreconf: running: gnu-configize
  gnu-configize: 'configure.ac' or 'configure.in' is required
  autoreconf: gnu-configize failed with exit status: 1

We're not pulling in the ext/tinydtls submodule, so this fails.

Upstream-Status: Inappropriate [oe-specific]
Signed-off-by: Alex Kiernan <alex.kiernan@gmail.com>
Signed-off-by: Alex Kiernan <alexk@zuma.ai>
---
 configure.ac | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/configure.ac b/configure.ac
index 9f51f4c67557..559808e03aa2 100644
--- a/configure.ac
+++ b/configure.ac
@@ -472,19 +472,6 @@ if test "x$build_dtls" = "xyes"; then
         have_tinydtls="no" # don't confuse AC_MSG_RESULT at the end of the script
     fi
 
-    # The user wants to use explicit TinyDTLS if '--with-tinydtls was set'.
-    if test "x$with_tinydtls" = "xyes" ; then
-        if test -d "$srcdir/ext/tinydtls"; then
-           AC_CONFIG_SUBDIRS([ext/tinydtls])
-           have_tinydtls="yes"
-         else
-           have_tinydtls="no" # don't confuse AC_MSG_RESULT at the end of the script
-         fi
-         have_gnutls="no" # don't confuse AC_MSG_RESULT at the end of the script
-         have_openssl="no" # don't confuse AC_MSG_RESULT at the end of the script
-         have_mbedtls="no" # don't confuse AC_MSG_RESULT at the end of the script
-    fi
-
     if test "$TLSCOUNT" -eq 0; then
       # The user hasn't requested the use of a specific cryptography library
       # we try first GnuTLS for usability ...
